racksdb/web/ui/src/components/BreadCrumbs.vue

```diff
@@ -0,0 +1,95 @@
+<!--Copyright (c) 2022-2023 Rackslab
+
+This file is part of RacksDB.
+
+SPDX-License-Identifier: GPL-3.0-or-later -->
+
+<script setup lang="ts">
+import { ChevronRightIcon } from '@heroicons/vue/24/outline'
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+console.log(route.name)
+
+const props = defineProps({
+  item: {
```

Comment:
Is there any good reason to not use the route meta entry right in this component instead of passing this item property?

Reply:
I used item property and not meta entry because i forgot i could use it but I've made the change.